OpenGL/Constructs/VAO.cs

```diff
@@ -432,11 +433,21 @@ public void Init(IGenericVBO[] vbos)
 
                 Draw = DrawOGL3;
                 DrawInstanced = DrawInstancedOGL3;
+
+                if (Gl.Version() >= 4 && Gl.VersionMinor() >= 3)
```

Comment:
This could break on OpenGL Version 5.0 (if that happens).  Maybe be something like:
```
if (Gl.Version() > 4 || (Gl.Version() == 4 && Gl.VersionMinor() >= 3))
```

Reply:
Curious if you had a chance to take a look at this?  I could also add this modification and merge if you'd prefer.